modules/react/_examples/stories/examples/SidePanelWithNavigation.tsx

```diff
@@ -0,0 +1,130 @@
+  {value: 'Key Lime Cheesecake', linkValue: '#KeyLimeCheesecake'},
+  {value: 'Tiramisu', linkValue: '#Tiramisu'},
+];
+
+const gettingStarted = [
+  'Extended Systems',
+  'Canvas Glossary',
+  'Frequently Asked Questions',
+  'Canvas Support',
+];
+
+export const WithNavigation = () => {
+  const {labelProps} = useSidePanel();
+
+  return (
+    <Flex height={800} position="relative" backgroundColor="frenchVanilla100">
+      <StyledPanel touched={true} variant="alternate">
+        <Flex as="ul" flexDirection="column" rowGap="zero" paddingInlineStart="zero">
+          <Flex.Item as="li" listStyle="none">
+            <Expandable padding="zero">
+              <StyledExpandable paddingTop="m" paddingBottom="xxs" paddingX="s" textAlign="center">
+                <SystemIcon icon={ribbonIcon} marginX="xxs" />
+                <Expandable.Title padding="zero">
+                  <BodyText size="small" fontWeight="bold" color="blackpepper300" marginY="zero">
+                    Bestsellers
+                  </BodyText>
+                  <Subtext size="small" color="blackpepper300" marginY="zero">
+                    Award winning sweet treats
+                  </Subtext>
+                </Expandable.Title>
+                <Expandable.Icon iconPosition="end" />
+              </StyledExpandable>
+              <Expandable.Content paddingY="zero" paddingX="zero" as="ul">
+                {bestsellers.map(item => {
+                  return (
+                    <li>
+                      <StyledLink
+                        size="large"
+                        display="flex"
+                        textDecoration="none"
+                        paddingLeft="xxl"
+                        paddingTop="xxs"
+                        paddingBottom="zero"
+                        href={item.linkValue}
+                      >
+                        {item.value}
+                      </StyledLink>
+                    </li>
+                  );
+                })}
+              </Expandable.Content>
+            </Expandable>
+          </Flex.Item>
+          <Flex.Item as="li" listStyle="none">
+            <Expandable padding="zero">
+              <StyledExpandable paddingTop="m" paddingBottom="xxs" paddingX="s" textAlign="center">
+                <SystemIcon icon={birthdayIcon} marginX="xxs" />
+                <Expandable.Title padding="zero">
+                  <BodyText size="small" fontWeight="bold" color="blackpepper300" marginY="zero">
+                    Custom Cakes
+                  </BodyText>
+                  <Subtext size="small" color="blackpepper300" marginY="zero">
+                    To celebrate all the milestones in life!
+                  </Subtext>
+                </Expandable.Title>
+                <Expandable.Icon iconPosition="end" flex="1" />
+              </StyledExpandable>
+              <Expandable.Content paddingY="zero" paddingX="zero">
```

Comment:
Apologies for missing this earlier in the morning! This line is missing as="ul" prop.